yara/snippets/yara.json

```
@@ -44,9 +44,50 @@
     },
     "Mach-O Header": {
         "prefix": "header_macho",
-        "body": "uint32(0) == 0xFEEDFACF ",
+        "body": "(uint16(0) == 0xfeca or uint16(0) == 0xfacf) ",
         "description": "Generate a condition to check for a Mach-O file header"
     },
+    "LNK Header": {
+        "prefix": "header_LNK",
+        "body": "uint16(0) == 0x004c and uint32(4) == 0x00021401 ",
+        "description": "Generate a condition to check for a LNK file header"
+    },
+    "Doc Header": {
+        "prefix": "header_Doc",
```

```suggestion
        "prefix": "header_all_Doc",
```
It might be useful to explicitly prefix this as a multi-header condition since this is a lot more comprehensive than the others. I think many people (myself included) would expect a single "doc" header check to be `0xd0cf11e0`

What do you think of that? Am I missing anything?